usr.sbin/Makefile

```diff
@@ -184,8 +184,13 @@ SUBDIR.${MK_OPENSSL}+=	keyserv
 SUBDIR.${MK_OPENSSL_KTLS}+=	rpc.tlsclntd
 SUBDIR.${MK_OPENSSL_KTLS}+=	rpc.tlsservd
 SUBDIR.${MK_PF}+=	ftp-proxy
-SUBDIR.${MK_PKGBOOTSTRAP}+=	pkg
+.if ${MACHINE_ABI:Mpurecap}
+# Install pkg64 and pkg64c with pkg.sh instead of pkg if world is purecap.
 SUBDIR.${MK_PKGBOOTSTRAP}.${MK_LIB64}+=	pkg64
+SUBDIR.${MK_PKGBOOTSTRAP}+=	pkg64c
```

Comment:
I don't understand why putting this here rather than in usr.sbin/pkg/Makefile is an improvement

Reply:
`usr.sbin/Makefile` already installs multiple programs depending on a world configuration.

I think it's better to review a single list in `usr.sbin/Makefile` to find out what programs are installed instead of reviewing program-specific Makefiles. It would be especially confusing in case of pkg64c: we would unconditionally install `usr.sbin/pkg64c` that becomes `pkg` if world is hybrid while `usr.sbin/pkg` exists but would be ignored.